api/conf/settings/logging.py

```diff
@@ -140,7 +143,8 @@ def suppress_unwanted_logs(record: LogRecord) -> bool:
     + shared_processors
     + [
         structlog.stdlib.ProcessorFormatter.wrap_for_formatter,
-    ],
+    ]
+    + [SentryProcessor(event_level=logging.ERROR)],
```

Comment:
This needs to come before the shared processors otherwise `exc_info` won't work:

> When you want to use structlog's built-in [format_exc_info](http://www.structlog.org/en/stable/api.html#structlog.processors.format_exc_info) processor, make that the SentryProcessor comes before format_exc_info! Otherwise, the SentryProcessor won't have an exc_info to work with, because it's removed from the event by format_exc_info.